src/xlsx/conversion/formula_conversion.ts

```diff
@@ -66,8 +66,8 @@ function convertFormula(formula: string, data: XLSXImportData): string {
     externalRefId = Number(externalRefId) - 1;
     cellRef = cellRef.replace(/\$/g, "");
 
-    const sheetIndex = data.externalBooks[externalRefId].sheetNames.findIndex(
-      (name) => name === sheetName
+    const sheetIndex = data.externalBooks[externalRefId].sheetNames.findIndex((name) =>
```

Comment:
I don't really know how the external references work, but it feels like it requires a test